### PR TITLE
Fix receive updates button style

### LINKED ISSUE #4: Add Hover State to - Receive Updates Button
The receive updates button currently doesn't have a hover state but should be updated to appear different on hover. Also, the link in the button should no longer have an underline (regardless of hover state). 

Screen Shot 2020-06-21 at 12 12 38 AM

### PR DESCRIPTION
**Pull request by omaratta212**
# Description

#### 1. Replaced the nested button with just an anchor tag

Previously it was:
```
<a href="#top">
    {" "}
    <TryItButton>Receive Updates</TryItButton>
</a> 
```
Now it's:
 ```
<TryItButton as="a" href="#top">Receive Updates</TryItButton>
```
**Why?**
1. Previously we were rendering a button tag inside an anchor tag which has [some problems](https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5) & I think it affects accessibility.
2. Now we can give the anchor tag `text-decoration: none` to get rid of the underline, which can't be done on a button tag & will require some inline styling.

#### 2. Changed the hover state styling for the button
Upon hovering, the button will be elevated 1px & the background will change to the theme accent colour with a transition, I think also we can add specific hover colour to the theme. let me know your thoughts about that.

## Related Issues
  Fixes #4 

## Have unit tests been added or modified? If not, why not?
- [ ] yes
- [x] no

I don't think it's something to test, right?

## Pages/Interfaces that will change
Home page
#### Before
Before

#### After
After

## Steps to test
1. Just open the homepage!

### CODE REVIEW

File: src/components/sections/getstarted.js [hunk 1]
```diff
@@ -60,7 +56,13 @@ const TryItButton = styled.button`
   border-color: initial;
   border-image: initial;
   outline: 0px;
+  display: inline-flex;
+  align-items: center;
+  text-decoration: none;
+  transition: all 0.2s ease;
   &:hover {
+    transform: translateY(-1px);
+    background: ${props => props.theme.color.accent};
```
Comment (M0nica) [inline src/components/sections/getstarted.js:65]: Were you proposing that a new color value be added to the `theme` file for the `hover` state that has the same value as `accent`? If so, that sounds good to me as in the future the `accent` color and `hover` color may diverge.

**Comment by M0nica** [comment]
This can be merged once the theme file is updated.

**Comment by M0nica** [comment]
Also FYI, I renamed `src/components/sections/getstarted.js` to `src/components/sections/getstarted.jsx` hence the merge conflict.

File: src/components/sections/getstarted.js (hunk 1 above)
Comment (omaratta212) [reply src/components/sections/getstarted.js:65]: Exactly, great I'll update the PR

**Comment by M0nica** [comment]
This looks great. Thanks!

**Pull request merged**